drafts/2019-03-22-async-rest-jaxrs-reactive-extensions.adoc

```diff
@@ -11,12 +11,17 @@ blog_description: "Combining reactive programming with JAX-RS 2.1 reactive exten
 = JAX-RS 2.1 Reactive Extensions with RxJava Backpressure
 John Koehler <https://github.com/jkoehler22>
 
-Combining reactive programming with JAX-RS 2.1 reactive extensions make for a powerful asynchronous processing that avoids overwhelming the consumer.   Let's see how.  
+Combining reactive programming with JAX-RS 2.1 reactive extensions make for a powerful asynchronous processing that avoids overwhelming the consumer.   Before we jump into a sample, let's define some of the basic concepts.  
+`Reactive programming` is an asynchronous programming paradigm that deals with data streams and the propagation of change. 
+`RxJava` is a `Reactive Extensions` JVM implementation that enables the writing of event-based, and asynchronous programs.  
+`Backpressure` is when in a processing pipeline, some asynchronous consumers have a build-up and can't process the items fast enough.  
+Let's see how all of these concepts work together.  
 We'll follow a sample data collection and retrieval application at:
 https://github.com/OpenLiberty/sample-reactive-extensions
-The application makes use of RxJava's `Flowable` to control the maximum amount of data collected for a given customer.
+The application data collection is accomplished with a simple loop emitting `Integer` items.  The application data retrieval is accomplished with a reactive extension asynchronous `get`.  The application makes use of RxJava's `Flowable` with `backpressure` to control the maximum amount of data collected for a given customer.  Flowable is an emitter of items that implements a back-pressure protocol.  Flowable was introduced in RxJava2.  Flowable comes into play when huge numbers of items are emitted that can not be handled by the consumer.  
```

Comment:
In most places, you use "backpressure" (all one word, no hyphen), but on this line there is "implements a back-pressure protocol" (with a hyphen).  Either way (hyphen or no hyphen) works for me, but I think we should be consistent.